### PR TITLE
ElementWiseStride==1 cases for legacy random ops 

### PR DESCRIPTION
**Pull request by quickwritereader**
## What changes were proposed in this pull request?

ElementWiseStride==1 cases for legacy random ops were added.

## How was this patch tested?

unit tests

## Quick checklist

The following checklist helps ensure your PR is complete:

- [x] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [x] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [x] Relevant tests for your changes are passing.

### CODE REVIEW

File: libnd4j/include/loops/cpu/random.cpp [hunk 1]
```diff
@@ -169,15 +183,27 @@ namespace functions {
...
-                auto func = PRAGMA_THREADS_FOR {
-                    PRAGMA_OMP_SIMD
-                    for (uint64_t i = start; i < stop; i += increment)  {
-                        auto offset = shape::indexOffset(i, xShapeInfo, xShapeInfoCast, canCastX);
-                        z[offset] = OpClass::op(x[offset], i, length, rng, extraArguments);
-                    }
-                };
-
-                samediff::Threads::parallel_for(func,  0, length, 1);
+                if(shape::elementWiseStride(zShapeInfo) ==  1){
```
Comment (raver119) [inline libnd4j/include/loops/cpu/random.cpp]: in this case you must also check xShapeInfo, and ensure their orders do match

File: libnd4j/include/loops/cpu/random.cpp [hunk 2]
```diff
@@ -56,18 +57,31 @@ namespace functions {
...
-                auto func = PRAGMA_THREADS_FOR {
-                    PRAGMA_OMP_SIMD
-                    for (auto i = start; i < stop; i += increment)  {
-                        auto offset = shape::indexOffset(i, xShapeInfo, xShapeInfoCast, canCastX);
-                        z[offset] = OpClass::op(x[offset], y[offset], i, length, rng, extraArguments);
-                    }
-                };
-
-                samediff::Threads::parallel_for(func,  0, length, 1);
+                if(shape::elementWiseStride(zShapeInfo) ==  1){
```
Comment (raver119) [inline libnd4j/include/loops/cpu/random.cpp]: in this case you must also check xShapeInfo and yShapeInfo, and ensure their orders do match

**Review by raver119 — CHANGES_REQUESTED** [review]
Fixes required.

**Comment by quickwritereader** [comment]
> Fixes required.

I added missing checks

**Review by raver119 — APPROVED** [review]

**Pull request merged**